config.toml

```diff
@@ -41,14 +41,9 @@ link = "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.cs
 # Flowbite
 link = "https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.5/flowbite.min.css"
 
-[[params.plugins.css]]
-# HighlightJS
-link = "https://unpkg.com/@highlightjs/cdn-assets/styles/default.min.css"
-
 ## JavaScript Plugins
-[[params.plugins.js]]
-# Lunr
```

Comment:
This was loaded in the `head` for each page. This plugin is currently only used on the `Data Playground` page, so I moved it. If we decided to use `Lunr` to make the whole site searchable, we can add it back here.